css/styles.css

```diff
@@ -1,19 +1,32 @@
-
-body {
-    height: 100vh;
-    width: 100vw;
-    margin: 0px;
+/* layout styles*/
+.content {
+    margin-top: 8%;
```

Comment:
you probably noticed that if you set your navbar to `fixed`, it overlaps with your content in an unexpected way. this is a quick and dirty fix, where we add an extra margin at the top of the content that is exactly the same as the height of the navbar.